Maple2.Server.Game/Model/Field/Actor/FieldActor.cs

```diff
@@ -21,12 +24,15 @@ internal sealed class FieldActor : IActor {
     public Transform Transform { get; init; }
     public BuffManager Buffs { get; }
     public Stats Stats { get; }
+    public AnimationState AnimationState { get; init; }
 
-    public FieldActor(FieldManager field) {
+    public FieldActor(FieldManager field, NpcMetadataStorage npcMetadata) {
         Field = field;
         Stats = new Stats(0, 0);
         Buffs = new BuffManager(this);
         Transform = new Transform();
+        NpcMetadata = npcMetadata;
+        AnimationState = new AnimationState(this, string.Empty); // not meant to have animations
```

Comment:
sorry, I meant instead of passing in a second argument, make the constructor have the default model name as `string.Empty`